examples/app-secaggplus/server.py

```diff
@@ -0,0 +1,46 @@
+import flwr as fl
+from flwr.common import Context
+from flwr.server import Driver, LegacyContext
+from flwr.server.workflow import SecAggPlusWorkflow
+from workflow_with_log import SecAggPlusWorkflowWithLogs
+
+
+# Define strategy
+strategy = fl.server.strategy.FedAvg(
+    fraction_fit=1.0,  # Select all available clients
+    fraction_evaluate=0.0,  # Disable evaluation
+    min_available_clients=3,
+    fit_metrics_aggregation_fn=lambda _: {},
```

Comment:
Do we need this?

Reply:
For demonstration purposes, evaluation is disabled. This follows the `app-pytorch` example.

Reply:
if we don't use eval, we can remove the entire line